src/server/registry/util/util_test.go

```diff
@@ -27,3 +29,185 @@ func TestIndexString(t *testing.T) {
 	assert.True(t, IndexString(a, "Z") == -1)
 	assert.True(t, IndexString(a, "") == -1)
 }
+
+func Test_pickItems(t *testing.T) {
```

Comment:
TestPickItems

Reply:
This test function name and skeleton are generated by the vscode-go, so let's keep this name.